### ISSUE TITLE
"Warning: Unknown prop `language` on <pre> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop     in pre (created by Highlight)

### ISSUE THREAD

**Issue by paulkogel** [bvaughn/react-highlight.js#6]
Full error message from my JS console when using this component:

```
"Warning: Unknown prop `language` on <pre> tag.
Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
in pre (created by Highlight)
```

**Comment by bvaughn** [comment]
Hey @paulkogel. Thanks for reminding me. Haven't used this library recently. :)

**Comment by bvaughn** [comment]
This warning should be fixed in 1.0.5.

**Comment by paulkogel** [comment]
great, thanks for fixing!
